src/main/java/ru/sbt/mipt/oop/home/Room.java

```diff
@@ -0,0 +1,63 @@
+package ru.sbt.mipt.oop.home;
+
+import ru.sbt.mipt.oop.door.Door;
+import ru.sbt.mipt.oop.event_handlers.DoorEventHandler;
+import ru.sbt.mipt.oop.event_handlers.LightEventHandler;
+import ru.sbt.mipt.oop.event_handlers.SensorEvent;
+import ru.sbt.mipt.oop.light.Light;
+
+import java.util.Collection;
+
+public class Room {
+
+    private Collection<Light> lights;
+    private Collection<Door> doors;
+    private String name;
+
+    public Room(Collection<Light> lights, Collection<Door> doors, String name) {
+        this.lights = lights;
+        this.doors = doors;
+        this.name = name;
+    }
+
+    public Collection<Light> getLights() {
+        return lights;
+    }
+
+    public Collection<Door> getDoors() {
+        return doors;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void handleLightEvent(SensorEvent event) {
```

Comment:
https://github.com/iagoshkov/smart-home-2020/blob/master/STUDENT_GUIDE.md#%D0%B4%D0%B5%D0%BB%D0%B5%D0%B3%D0%B8%D1%80%D1%83%D0%B9%D1%82%D0%B5-%D0%BE%D0%B1%D1%80%D0%B0%D0%B1%D0%BE%D1%82%D0%BA%D1%83-%D1%81%D0%BE%D0%B1%D1%8B%D1%82%D0%B8%D0%B9-%D0%BE%D1%82%D0%B4%D0%B5%D0%BB%D1%8C%D0%BD%D1%8B%D0%BC-%D0%BA%D0%BB%D0%B0%D1%81%D1%81%D0%B0%D0%BC